supervisors/sensitivity_analysis_morris/morris.R

```diff
@@ -0,0 +1,107 @@
+require(methods)
+library("rjson")
+library("sensitivity")
+library("httr")
+
+source("supervisors/sensitivity_analysis_morris/scalarmapi.R")
```

Comment:
Nie zostawiajmy rzeczy do zrobienia na "potem", bo "potem" === "nigdy".

File.dirname(__FILE__) to aktualna ścieżka pliku. source(Dir[path].first) poeinno zadziałać, jeżeli posiadasz pożądaną ścieżkę, w tym przypadku pewnie "#{File.dirname(__FILE__}/scalarmapi.R", albo bardziej uniwersalnie "#{File.dirname(__FILE__}/**/scalarmapi.R". ** oznacza dowolny kawałek ścieżki - dowolne katalogi lub nic. 

Gdybyśmy chcieli całkiem uniezależnić się od położenia, można zrobić ścieżkę Rails.root.join('**', 'scalarmapi.R').to_s, ale możemy trafić na więcej plików o tej nazwie. Rails.root.join to po prostu doklejanie do nadrzędnego katalogu całego serwisu.

Mam nadzieję, że wyczerpałem temat. Wszystkie wklejki z kodu są pisane na kolanie, zanim czegoś użyjecie to sprawdźcie bo nie odpowiadam za poprawność.